org.simondalvai.ball2box.appdata.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.simondalvai.ball2box</id>
+  <name>Ball2Box</name>
+  <developer_name>Simon Dalvai</developer_name>
+  <launchable type="desktop-id">org.simondalvai.ball2box.desktop</launchable>
+  <project_license>AGPL-3.0-or-later</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>A casual game with the Godot engine 3.</summary>
+  <url type="homepage">https://simondalvai.org/games/ball2box</url>
+  <url type="bugtracker">https://github.com/dulvui/ball2box/issues</url>
+  <url type="donation">https://simondalvai.org/donate</url>
+  <url type="vcs-browser">https://github.com/dulvui/ball2box<</url>
+  <description>
+    <p>Swipe to shoot the ball into the box in over 100 levels with only one finger. Can you get
+      all three stars on all levels?
+    </p>
+    <ul>
+      <li>🕹️ 100+ levels and more coming soon...</li>
+      <li>🌈 20+ different styled balls to collect</li>
+      <li>📡 Offline game</li>
+      <li>🕵️‍♀️ Privacy friendly</li>
+      <li>👨‍💻 Created with Godot Engine 3</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      https://raw.githubusercontent.com/dulvui/ball2box/main/metadata/en-US/images/phoneScreenshots/Android-1.png</screenshot>
+    <screenshot>
+      https://raw.githubusercontent.com/dulvui/ball2box/main/metadata/en-US/images/phoneScreenshots/Android-2.png</screenshot>
+    <screenshot>
+      https://raw.githubusercontent.com/dulvui/ball2box/main/metadata/en-US/images/phoneScreenshots/Android-3.png</screenshot>
+    <screenshot>
+      https://raw.githubusercontent.com/dulvui/ball2box/main/metadata/en-US/images/phoneScreenshots/Android-4.png</screenshot>
+    <screenshot>
+      https://raw.githubusercontent.com/dulvui/ball2box/main/metadata/en-US/images/phoneScreenshots/Android-5.png</screenshot>
+    <screenshot>
+      https://raw.githubusercontent.com/dulvui/ball2box/main/metadata/en-US/images/phoneScreenshots/Android-6.png</screenshot>
+    <screenshot>
+      https://raw.githubusercontent.com/dulvui/ball2box/main/metadata/en-US/images/phoneScreenshots/Android-7.png</screenshot>
+    <screenshot>
+      https://raw.githubusercontent.com/dulvui/ball2box/main/metadata/en-US/images/phoneScreenshots/Android-8.png</screenshot>
```

Comment:
Inside image tags https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#screenshots

Reply:
Also 

>  If the images are hosted in a git repository, the link should be from a tag or a commit and not a branch.

Reply:
using images from commit now

Reply:
it's not in https://raw.githubusercontent.com/dulvui/ball2box/aa9cca3da89a20f7c15f163453864d9b5e9cc6f9/flatpak/org.simondalvai.ball2box.appdata.xml


that's the build failure.